Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -851,6 +851,8 @@ def setup_(self, step, stepName, stepDict, k, properties):
               stepDict[stepName][k] = None
             else:
               stepDict[stepName][k] = merge([self.__reco, stepDict[step][k]])
+            if 'Phase2' in stepDict[stepName][k]['--era'] and 'DQM:@standardDQM' in stepDict[stepName][k]['-s']:
+              stepDict[stepName][k]['-s'] = stepDict[stepName][k]['-s'].replace('DQM:@standardDQM','DQM:@phase2')
```

Comment:
Thx @AdrianoDee 

To replace only `DQM:@standardDQM`? I assume we should try to make it the same as standard Phase-2 workflow, for example, `VALIDATION:@standardValidation` ==> `VALIDATION:@phase2Validation`, to avoid failure in Validation of HLT.

Patatrack:
`VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@ExtraHLT+@miniAODDQM+@pixelTrackingOnlyDQM`

Phase2:
`VALIDATION:@phase2Validation+@miniAODValidation,DQM:@phase2+@miniAODDQM`

Reply:
But maybe keep `@pixelTrackingOnlyDQM`. Not sure about `ExtraHLT`, but I think for Phase-2, we don't need it for now.

Reply:
Indeed `@pixelTrackingOnlyDQM` should work with Phase2